.buildkite/pipeline.yml

```diff
@@ -1024,23 +1024,23 @@ steps:
       - label: ":fire: Flame graph: perf target (Callbacks)"
         command: >
           julia --color=yes --project=perf perf/flame.jl
-          --config_file  $PERF_CONFIG_PATH/flame/perf_target_callbacks.yml
+          flame_perf_target_callbacks
         artifact_paths: "flame_perf_target_callbacks/*"
         agents:
           slurm_mem: 20GB
 
       - label: ":fire: Flame graph: gravity wave"
         command: >
           julia --color=yes --project=perf perf/flame.jl
-          --config_file $PERF_CONFIG_PATH/flame/perf_gw.yml
+          flame_perf_gw
         artifact_paths: "flame_perf_gw/*"
         agents:
           slurm_mem: 20GB
 
       - label: ":fire: Flame graph: perf target (diagnostics)"
         command: >
           julia --color=yes --project=perf perf/flame.jl
-          --config_file $PERF_CONFIG_PATH/flame/diagnostics.yml
+          flame_perf_diagnostics
```

Comment:
So is this just using ARGS now?

Reply:
Yes, but we could use argparse. Do you think that would be clearer?

Reply:
The only benefit I see is that it's keyword based rather than positional. It's not a big issue if there's a small number of arguments, but 🤷🏻